static/js/jquery.js

```diff
@@ -0,0 +1,3 @@
+import jquery from 'jquery';
+
+export default (window.$ = window.jQuery = jquery);
```

Comment:
Shouldn't this be defined in `static/index.js` directly?

Reply:
Well, it could work, but from what i read on the subject Parcel isn't really good with this kind of global var, and this is an recommended hack.

Reply:
Maybe just add a comment with the link to the hack/snippet ;)